Italian translation updated

erciccione opened the pull request:
Ok, this time everything should be fine.     
- Typos edited
- updated many "unfinished" lines
- edited plurals and little mistakes (thanks @jonathancross )

I left a couple of "unfinished" lines because i'm not sure about the translation in that specific context

jonathancross commented:
@erciccione This needs to be rebased as the translations have been updated.
Please let me know if you are comfortable doing that or want me to help out?

erciccione commented:
@jonathancross Thank you so much fo ryour help! I rebased, solved the conflicts and squashed all the commits in one, but i can't squash the "merge branch 'master' into master" commit . is this happening because i edited the confilcts directly on github?

jonathancross commented:
Yeah, unfortunately GitHub has limited git functionality, this (rebasing and squashing commits) is one area it doesn't do well in.

jonathancross commented on translations/monero-core_it.ts:
```diff
@@ -862,12 +838,12 @@
         <location filename="../pages/Receive.qml" line="344"/>
         <source>&lt;p&gt;&lt;font size=&apos;+2&apos;&gt;This is a simple sales tracker:&lt;/font&gt;&lt;/p&gt;&lt;p&gt;Click Generate to create a random payment id for a new customer&lt;/p&gt; &lt;p&gt;Let your customer scan that QR code to make a payment (if that customer has software which supports QR code scanning).&lt;/p&gt;&lt;p&gt;This page will automatically scan the blockchain and the tx ...
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">&lt;p&gt;&lt;font size=&apos;+2&apos;&gt;Questo è un semplice tracker per le vendite:&lt;/font&gt;&lt;/p&gt;&lt;p&gt;Clicca "Genera" per creare un ID Pagamento casuale per un nuovo cliente&lt;/p&gt; &lt;p&gt;Fai scannerizzare al cliente il codice QR per effettuare un pagamento (Se il cliente possiede un QR scanner).&lt;/p&gt;&lt;p&gt;Questa pagina scannerizz ...
```
`type="unfinished"` should be removed.

jonathancross commented on translations/monero-core_it.ts:
```diff
@@ -1815,14 +1805,14 @@ Ringsize: </source>
     <name>WizardCreateWallet</name>
     <message>
         <source>A new wallet has been created for you</source>
-        <translation type="vanished">Un nuovo portafoglio è stato creato per te</translation>
```
We should probably remove translations that have vanished.

jonathancross commented on translations/monero-core_it.ts:
```diff
@@ -519,107 +519,87 @@
         <location filename="../pages/Mining.qml" line="164"/>
         <source>Mining is only available on local daemons. Run a local daemon to be able to mine.&lt;br&gt;</source>
-        <translation type="unfinished">Minare è solo possibile utilizzando il daemon locale. Avvia daemon locale per poter minare</translation>
+        <translation type="unfinished">Minare è possibile solo utilizzando il daemon locale. Avvia daemon locale per iniziare a minare&lt;br&gt;</translation>
```
`type="unfinished"` should be removed.

jonathancross commented on translations/monero-core_it.ts:
```diff
@@ -1325,45 +1302,47 @@
     </message>
     <message>
         <location filename="../components/TableDropdown.qml" line="183"/>
+        <source>&lt;b&gt;Send to same destination&lt;/b&gt;</source>
```
Looks like you have an error here (should only be 1 source and 1 translation).
The ones below ("**Send to this address**") are correct per [#685](https://github.com/monero-project/monero-core/pull/685/files)

jonathancross commented on translations/monero-core_it.ts:
```diff
@@ -1477,41 +1456,38 @@ Please upgrade or connect to another daemon</source>
         <location filename="../pages/Transfer.qml" line="175"/>
         <source>All</source>
+        <translation>Tutto</translation>
         <translation type="unfinished">Tutte</translation>
```
`type="unfinished"` should be removed.

jonathancross commented on translations/monero-core_it.ts:
```diff
@@ -1477,41 +1456,38 @@ Please upgrade or connect to another daemon</source>
     <message>
         <location filename="../pages/Transfer.qml" line="44"/>
         <source>&lt;style type=&apos;text/css&apos;&gt;a {text-decoration: none; color: #FF6C3C; font-size: 14px;}&lt;/style&gt;&lt;font size=&apos;2&apos;&gt; (&lt;/font&gt;&lt;a href=&apos;#&apos;&gt;Start daemon&lt;/a&gt;&lt;font size=&apos;2&apos;&gt;)&lt;/font&gt;</source>
         <translation type="unfinished"></translation>
```
Would be nice to have a translation for "Start daemon" here.
(hard to read with all the escaped html and css code ;-)

erciccione replied:
I don't think we need a translation for that. "Start" is one of those word that we use aswel ;)

jonathancross replied:
Okay, we might as well copy / paste the text below then and remove `type="unfinished` so people know this is intentionally left as English.

jonathancross commented on translations/monero-core_it.ts:
```diff
@@ -140,37 +140,37 @@
     <name>DaemonProgress</name>
     <message>
         <source>Synchronizing blocks %1/%2</source>
-        <translation type="vanished">Sincronizzazione blocchi %1/%2</translation>
```
I think translations `type="vanished"` should be removed.

jonathancross replied:
I was suggesting the whole translated message be removed because it is no longer needed.
Same with the other `type="vanished"` below.

jonathancross commented on translations/monero-core_it.ts:
```diff
@@ -1147,11 +1123,12 @@
     </message>
     <message>
         <source>Daemon adress</source>
-        <translation type="vanished">Indirizzo daemon</translation>
```
I think translations `type="vanished"` should be removed.

jonathancross replied:
See note about `type="vanished"`

jonathancross commented on translations/monero-core_it.ts:
```diff
@@ -2093,18 +2083,14 @@ Ringsize: </source>
         <translation type="vanished">E&apos; veramente importante che ti salvi queste parole in quanto sono l&apos;unico backup che serve per il tuo portafoglio. Ti sarà chiesto di confermarle nella prossima schermata per essere sicuri che tu le abbia copiate correttamente.</translation>
```
I think translations `type="vanished"` should be removed.

jonathancross commented on translations/monero-core_it.ts:
```diff
@@ -2209,19 +2195,19 @@ Ringsize: </source>
     <name>WizardRecoveryWallet</name>
     <message>
         <source>My account name</source>
-        <translation type="vanished">Nome del mio account</translation>
```
I think translations `type="vanished"` should be removed.

erciccione commented:
- Removed all "vanished" and "unfinished" lines as signaled
- Corrected mistakes

Thanks @jonathancross !! The only bad side is that now i have 3 unsquashable commits. Hope @fluffypony is not going to hate me too much.

This should be good to go now

erciccione commented:
Ok @jonathancross , i finally stashed all the commits in one and integrated your changes. I have also deleted all the obsolete lines left and checked all the 'unfinished' lines. Now everything should be fine for real. Thank you again for reviewing this :)

jonathancross commented:
Almost there  :-)
I discovered an XML error and sent the fix in this PR: https://github.com/erciccione/monero-core/pull/1

erciccione commented:
missed that one! merged. I own you at least a couple of beers ;)

jonathancross commented:
@fluffypony This is ready to merge.
